packages/desktop/views/dashboard/governance/components/ProposalCard.svelte

```diff
@@ -43,7 +46,16 @@
         <Text truncate>{proposal.title}</Text>
     </div>
     <div class="flex justify-between items-center">
-        <ProposalStatusInfo {proposal} />
+        <div class="flex items-center gap-2">
+            <ProposalStatusInfo {proposal} />
+            {#if proposal?.status === EventStatus.Holding}
```

Comment:
The remaining time should also show for all other event statuses (except ended).
Upcoming -> Time until commencing
Commencing -> Time until holding
Holding -> Time until ended

![image](https://github.com/bloomwalletio/bloom/assets/45079109/e59f50ad-31d8-454e-a6d7-2f5c87b4a553)